src/main/java/com/technicjelle/Cheese.java

```diff
@@ -79,62 +55,48 @@ public Collection<Shape> getHoles() {
 		return holes;
 	}
 
-	/**
-	 * Create a BlueMap {@link Shape} with potentially some holes, from a collection of chunks.<br>
-	 * Designed to be fed directly into a BlueMap {@link ShapeMarker} or {@link ExtrudeMarker}
-	 *
-	 * @param chunks The chunks to create the cheese from
-	 * @return A cheese created from the given chunks, if possible
-	 * @throws InvalidSelectionException If the chunks are not connected
-	 * @see #createFromCells(Vector2d, Vector2i...)
-	 * @see #createMultiCheeseFromChunks(Vector2i...)
-	 */
-	public static Cheese createFromChunks(Vector2i... chunks) throws InvalidSelectionException {
-		if (!checkConnected(chunks))
-			throw new InvalidSelectionException("Chunks are not connected");
-
-		return createFromChunksUnsafe(chunks);
+	public static Collection<Cheese> createPlatterFromChunks(Vector2i... chunks) {
+		return createPlatterFromCells(CHUNK_CELL_SIZE, chunks);
 	}
 
-	/**
-	 * Same as {@link #createFromChunks(Vector2i...)}, but skips the connected check.<br>
-	 * This is useful if you are 100% sure that your chunks will be connected.
-	 *
-	 * @param chunks The chunks to create the cheese from
-	 * @return A cheese created from the given chunks
-	 */
-	public static Cheese createFromChunksUnsafe(Vector2i... chunks) {
-		return createFromCellsUnsafe(CHUNK_CELL_SIZE, chunks);
+	public static Cheese createSingleFromChunks(Vector2i... chunks) {
+		return createSingleFromCells(CHUNK_CELL_SIZE, chunks);
 	}
 
-	/**
-	 * Same as {@link #createFromChunks(Vector2i...)}, but allows you to specify your own cell-size,
-	 * instead of the default 16x16 for full chunks.<br>
-	 * This is useful in case your area data isn't chunk-based.
-	 *
-	 * @param cellSize The size of a cell
-	 * @param cells    The cells to create the cheese from
-	 * @return A cheese created from the given cells, if possible
-	 * @throws InvalidSelectionException If the cells are not connected
-	 * @see #createFromChunks(Vector2i...)
-	 * @see #createMultiCheeseFromCells(Vector2d, Vector2i...)
-	 */
-	public static Cheese createFromCells(Vector2d cellSize, Vector2i... cells) throws InvalidSelectionException {
-		if (!checkConnected(cells))
-			throw new InvalidSelectionException("Cells are not connected");
+	public static Collection<Cheese> createPlatterFromCells(Vector2d cellSize, Vector2i... cells) {
+		Set<Vector2i> remainingCells = new HashSet<>(Set.of(cells));
```

Comment:
Ok did anyone actually run the profiler on this one?
```suggestion
		Set<Vector2i> remainingCells = new HashSet<>(List.of(cells));
```

Before:
![27-155259](https://github.com/TechnicJelle/BMUtils/assets/45627874/4e689297-3194-4761-90e4-c22d6e0e7358)
![27-155313](https://github.com/TechnicJelle/BMUtils/assets/45627874/7ea6a73d-48cf-480f-86bf-49611e9c6497)
![27-155439](https://github.com/TechnicJelle/BMUtils/assets/45627874/068a0cdb-3ed2-4312-b20f-87903bedda8c)

After:
![27-155539](https://github.com/TechnicJelle/BMUtils/assets/45627874/520a06c6-3b1c-4c9c-af49-b0f88617fd1c)